wit/webgpu.wit

```diff
@@ -247,11 +255,10 @@ interface webgpu {
         label: func() -> string;
         set-label: func(label: string);
     }
-    resource remote-buffer {
-        // what about wasm64? Use u64?
-        length: func() -> u32;
-        get: func(i: u32) -> u8;
-        set: func(i: u32, val: u8);
+    // TODO: This should be replcated with something that doesn't require a copy.
+    resource temp-buffer {
```

Comment:
I see why this is named as such, but perhaps something like "copied buffer" could be better? Even if this ends up going away in the future, temporary could make this sound like it is a buffer with a temp lifetime, instead of a temporary shelf-life API

Reply:
How do you feel about `non-standard-buffer`? I wanted anyone that looks at it to immediately know that this is not going to be part of the standard

Reply:
Let's go with `non-standard-buffer`, and then we can document in the future when this changes to remove copies